devbin/generate_gcp_ar_cleanup_policy.py

```diff
@@ -12,15 +12,17 @@ def to_dict(self):
 
```

Comment:
I don't entirely understand why pyright runs on everything but it does and it dislikes the formatting of this file.